src/guide/blockchain/assets.md

```diff
@@ -0,0 +1,83 @@
+# Assets
+
+Iroha has been built with few underlying assumptions about what the assets
+need to be.
+
+The assets can be **fungible** (every £1 is exactly the same as every other
+£1) or **non-fungible** (a £1 bill signed by the Queen of Hearts is not the
+same as a £1 bill signed by the King of Spades).
```

Comment:
@s8sato I was under the same impression, but then realized that true NFTs should disallow instantiating more than one of an asset definition (`monalisa#louvre#alice@wland` and `monalisa#louvre#bob@wland` cannot co-exist since there is only one Mona Lisa in Louvre).

BTW, I think that Store assets, as designed right now, don't make much sense because two instances of the same definition can have totally different keys.

Reply:
Apart from that, as for this doc, what do you think about not mentioning to FT/NFT but just mentioning it is planned to be implemented

Reply:
I think we can omit NFT/FT and focus on the distinction between Numeric (with elastic or fixed global supply) and Store assets for now